kclvm/error/src/lib.rs

```diff
@@ -271,6 +354,20 @@ pub enum ParseError {
     UnexpectedToken { expected: Vec<String>, got: String },
 }
 
+impl fmt::Display for ParseError {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match self {
+            ParseError::UnexpectedToken { expected, got } => {
+                return write!(
+                    f,
+                    "Unexpected token, expected '{:?}', got '{}'",
```

Comment:
Here, token symbols such as `+`, `-`, `*` and `/` should be used instead of `{:?}` of the token structure format?